bin/node/inspect/src/command.rs

```diff
@@ -25,14 +25,12 @@ use crate::{
 use sc_cli::{CliConfiguration, ImportParams, Result, SharedParams};
 use sc_service::{Configuration, NativeExecutionDispatch};
 use sp_runtime::traits::Block;
-use std::str::FromStr;
 
 impl InspectCmd {
 	/// Run the inspect command, passing the inspector.
 	pub fn run<B, RA, D>(&self, config: Configuration) -> Result<()>
 	where
 		B: Block,
-		B::Hash: FromStr,
```

Comment:
Was it the case that this trait bound is only required in situations where `std` is enabled?

Reply:
yeah ，  there is maybefromstr

Reply:
I think you answered a different question: I was asking if it _was_ required in situations where `std` is enabled, i.e. before you introduced `MaybeFromStr`, not whether it is the case _right now_.

Reply:
You know that the runtime does not encourage the use of the `String` type. So `sp_std` do not export `str` mod.

Reply:
But I personally can't understand why this is done, it may be to prevent problems such as code bloat or utf-8 encoding  problems